### ISSUE TITLE
Bounding Box Color remains unchanged?

### ISSUE THREAD

**Issue by CJavier11** [ultralytics/ultralytics#814]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar bug report.

### YOLOv8 Component

Detection

### Bug

I have tried changing the hex values in the Colors class in plotting.py but the output of my bounding boxes remains the same.  I've also tried returning a dictionary correlating to each class number, which also fails to work. Is the color of the bounding box not affected by the plotting.py colors class?

class Colors:
    # Ultralytics color palette https://ultralytics.com/
    def __init__(self):
        # hex = matplotlib.colors.TABLEAU_COLORS.values()
        hexs = ('0000FF', 'FF0000', '800080', 'FFFF00', 'FF69B4', '00FF00', 'FFFFFF', '00FFFF', '0000FF', 'FFA500',
                '000000', '808080')
        self.palette = [self.hex2rgb(f'#{c}') for c in hexs]
        self.n = len(self.palette)

    def __call__(self, i, bgr=False):
        c = self.palette[int(i) % self.n]
        return (c[2], c[1], c[0]) if bgr else c

    @staticmethod
    def hex2rgb(h):  # rgb order (PIL)
        return tuple(int(h[1 + i:1 + i + 2], 16) for i in (0, 2, 4))

colors = Colors()  # create instance for 'from utils.plots import colors'
#colors = {0: (0, 0, 255), 1: (255, 0, 0), 2: (128, 0, 128), 3: (255, 255, 0), 4: (255, 105, 180), 5: (0, 255, 0), 
                6: (255, 255, 255), 7: (0, 255, 255), 8: (0, 0, 255), 9: (255, 165, 0), 10: (0, 0, 0), 11: (128, 128, 128)}

### Environment

Ultralytics YOLOv8.0.25 🚀 Python-3.8.10 torch-1.13.1+cu116 CUDA:0 (NVIDIA A100-SXM4-40GB, 40536MiB)

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by glenn-jocher** [comment]
@CJavier11 yes this is the correct and only place to modify colors. I did a quick test and modifications work, so removing bug label.

**Comment by CJavier11** [comment]
Here is an output image from the changes made to the Colors class above.  Below is also the classes from yaml I utilized to train the model.
<img width="491" alt="Screenshot 2023-02-05 at 4 05 57 PM" src="https://user-images.githubusercontent.com/90358358/216845773-373a0a16-ce59-4395-88dd-69c5e3ea2863.png">

# Classes
names:
  0: Blue Hold
  1: Red Hold
  2: Purple Hold
  3: Yellow Hold
  4: Pink Hold
  5: Green Hold
  6: White Hold
  7: Person
  8: Down Climb
  9: Orange Hold
  10: Black Hold
  11: Foot Chip
